tests/ovs_test.go

```diff
@@ -27,8 +27,8 @@ var _ = Describe("ovs-cni", func() {
 	Describe("pod availability tests", func() {
 		Context("pod availability tests", func() {
 			It("assert pods exists", func() {
-				pods, _ := clientset.CoreV1().Pods("").List(context.TODO(), v1.ListOptions{})
-				Expect(len(pods.Items)).Should(BeNumerically(">", 0))
+				pods, _ := clientset.CoreV1().Pods("").List(context.TODO(), metav1.ListOptions{ LabelSelector: "app=ovs-cni"})
```

Comment:
does this pass the syntax check ? (extra space in `ListOptions{-> <-LabelSelector: "app=ovs-cni"})`

Reply:
Done